packages/manager/apps/pci-load-balancer/src/routes.tsx

```diff
@@ -1,16 +1,20 @@
 import { lazy } from 'react';
-import { Route, Routes } from 'react-router-dom';
+import { Navigate, Route, Routes } from 'react-router-dom';
 
-const ROUTE_PATHS = {
+export const ROUTE_PATHS = {
```

Comment:
we will have to change routing back to using createHashRouter and defining routes in array because we spotted issues with ErrorBoundary not working with HashRouter component (not compatible with data api)